.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+jobs:
+  python-3.4:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.4
+
+    steps: &build
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "requirements.txt" }}
+
+      - run:
+          name: virtual-env
+          command: |
+            if [ ! -d "$HOME/.coala-venv" ]; then python3 -m venv ~/.coala-venv; fi
+            echo "source ~/.coala-venv/bin/activate" >> $BASH_ENV
+
+      - run:
+          name: pre
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install 8
+            nvm alias default node
+            npm install -g csslint
+
+      - run:
+          name: dependencies
+          command: |
+            pip install -r test-requirements.txt
+            pip install -r requirements.txt
+            pip install -r docs-requirements.txt
+
+      - run:
+          name: pytest
+          when: always
+          command: py.test
+
+      - run:
+          name: CLI-Test
+          when: always
+          command: |
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            pip install pip setuptools -U
+            python setup.py install
+            pip install coala-bears[alldeps] --pre -U
+            pip install -r docs-requirements.txt
+            coala --non-interactive
+
+      - run:
+          name: Codecov
```

Comment:
you switch between first letter upper and first letter lower - pick one!
Actually no, just use lower, as the objective of each section is to run a command, which use lower case.